telegram-bot/src/main/java/com/exadel/telegrambot/bot/bot/Bot.java

```diff
@@ -0,0 +1,16 @@
+package com.exadel.telegrambot.bot.bot;
+
+import com.exadel.telegrambot.bot.service.BotService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.telegram.telegrambots.meta.api.objects.Update;
+
+@Service
+@RequiredArgsConstructor
+public class Bot {
+    private final BotService botService;
+
+    public void updateHandler(Update update) {
```

Comment:
Why this method is empty?

Reply:
I am writing now it in this push I just fix configuration

Reply:
now you can see some changes